client/src/App.js

```diff
@@ -17,7 +17,7 @@ import Room from "./pages/Room";
 import Error from "./pages/Error";
 import Signup from "./pages/Signup";
 import Login from "./pages/Login";
-import Review from "./pages/Review";
```

Comment:
This one should be import Review from "./pages/Review"; instead of import Review from "./components/Reviews";
the review component is imported in the review page